components/dot_menu/dot_menu.test.tsx

```diff
@@ -77,42 +77,6 @@ describe('components/dot_menu/DotMenu', () => {
         expect(wrapper).toMatchSnapshot();
     });
 
-    test('should have divider when able to edit or delete post', () => {
-        const props = {
-            ...baseProps,
-            canEdit: true,
-            canDelete: true,
-        };
-        const wrapper = shallowWithIntl(
-            <DotMenu {...props}/>,
-        );
-
-        expect(wrapper.state('canEdit')).toBe(true);
-        expect(wrapper.state('canDelete')).toBe(true);
-        expect(wrapper.find('#divider_post_post_id_1_edit').exists()).toBe(true);
-
-        wrapper.setProps({isReadOnly: true});
-
-        expect(wrapper.state('canEdit')).toBe(false);
-        expect(wrapper.state('canDelete')).toBe(false);
-        expect(wrapper.find('#divider_post_post_id_1_edit').exists()).toBe(false);
-    });
-
-    test('should not have divider when able to edit or delete a system message', () => {
-        const props = {
-            ...baseProps,
-            post: TestHelper.getPostMock({
-                ...baseProps.post,
-                type: PostTypes.JOIN_CHANNEL as PostType,
-            }),
-        };
-        const wrapper = shallowWithIntl(
-            <DotMenu {...props}/>,
-        );
-
-        expect(wrapper.find('#divider_post_post_id_1_edit').exists()).toBe(false);
-    });
```

Comment:
Divider always appears now regardless of edit/delete permissions so tests no longer make sense